### PR TITLE
Call onScroll prop with scroll information if defined

### PR DESCRIPTION
**Pull request by maxhudson**

### CODE REVIEW

**Comment by maxhudson** [comment]
onscroll needs to pulled out to bound fn, but otherwise pretty much good to go

**Comment by mgramigna** [comment]
Thanks for the quick responses! Just addressed your comment.

**Comment by mgramigna** [comment]
Also I am not sure if you wanted me to include the corresponding `dist` files in this PR as a result of the build script. Let me know if you need me to remove it.

**Comment by maxhudson** [comment]
@mgramigna cool! I'll bump the version and merge it in the next few days.

File: src/index.js [hunk 1]
```diff
@@ -31,6 +31,21 @@ class StickyTable extends PureComponent {
     super(props);
 
     this.index = index = index + 1;
+    this.onScroll = this.onScroll.bind(this);
+  }
+
+  onScroll(e) {
+    if (this.props.onScroll) {
+      const { target } = e.nativeEvent;
```
Comment (sibelius) [inline src/index.js]: can't you do the same on user land instead?

**Comment by mgramigna** [comment]
Update on this:

I think for most use cases, you can do the same in user land, yes. The case that a coworker of mine was working with was a little unique and I think the testing we did in user land didn't work whereas having it on the sticky table itself did.

However, we have found another solution that actually does not require this to be a part of `react-sticky-table`, so feel free to close this PR if you don't actually want `onScroll` back in this library. No harm done.

**Comment by cameronbraid** [comment]
> Update on this:
> 
> I think for most use cases, you can do the same in user land, yes. The case that a coworker of mine was working with was a little unique and I think the testing we did in user land didn't work whereas having it on the sticky table itself did.
> 
> However, we have found another solution that actually does not require this to be a part of `react-sticky-table`, so feel free to close this PR if you don't actually want `onScroll` back in this library. No harm done.

I'm needing to tack scroll position, what solution did you end up using ?

**Comment by maxhudson** [comment]
Closing this specific PR as it's way out of date code-wise

**Comment by maxhudson** [comment]
@cameronbraid you should be able to just go `<StickyTable onScroll={(event) => console.log(event.scrollTop)} />` from the look of the code, but I haven't tested that

**Pull request closed without merging**